demo/midgard-node/src/commands/listen.ts

```diff
@@ -329,7 +329,8 @@ const getMergeHandler = Effect.gen(function* () {
 
 const getResetHandler = Effect.gen(function* () {
   yield* Effect.logInfo(`🚧 Reset request received`);
-  yield* StateQueueTx.resetStateQueue;
+  yield* Reset.resetAll;
+
   yield* Effect.all(
```

Comment:
Did you consider putting all these `.clear` calls under the `Reset` module as well? Then we'd be able to get `reset.ts` outside `/transactions` directory.